AdyenComponents/PayTo/PayToComponent.swift

```diff
@@ -198,6 +202,45 @@ public final class PayToComponent: PaymentComponent,
         return item
     }()
 
+    /// The  account holder email text input item.
+    internal lazy var emailInputItem: FormTextInputItem = {
+        let item = FormTextInputItem(style: configuration.style.textField)
+        // TODO: Add translation
+        item.title = localizedString(LocalizationKey(key: "Email"), configuration.localizationParameters)
```

Comment:
we have translation for `emailItemTitle` which is "Email address`. I think we can use this. Doesn't have to be just "email` imo.

Reply:
makes sense :)